shapely/geometry/base.py

```diff
@@ -67,6 +73,23 @@ class JOIN_STYLE:
     bevel = 3
 
 
+def empty_svg_doc_element():
+    doc = ET.Element('svg', xmlns=SVG_NAMESPACE)
+    doc.set('xmlns:xlink', SVG_XLINK_NAMESPACE)
+    return doc
```

Comment:
@jreiberkyle In my opinion this function isn't called in enough places to be justified. I subscribe to the rule of 3: duplication two times is fine, make a new function the third time.

My suggestion is to copy this into svg_doc_element and then for the empty case below, return `ET.Element('svg', xmlns=SVG_NAMESPACE)`.

Reply:
makes sense, will do!